### DISCUSSION TITLE
add layers for spooky kids fla files

### DISCUSSION THREAD

**Discussion by SPRINGEY** [FunkinCrew/Funkin#1673] [Ideas]
can you please add layers for skid and pumps flash files? every other character has a body part you can modify and make it apart of the full animation but for some reason for skid and pump that's not the case. so could you please add this?

**Comment by cuckydev** [comment]
I believe this is because Skid and Pump were animated using more frame-by-frame than tweening. I don't believe this can be changed.

**Comment by SPRINGEY** [comment]
could i like maybe add some if thats even possible

**Comment by ghost** [comment]
ok

**Comment by roythearsonist** [comment]
> could i like maybe add some if thats even possible

Sure, if you want. Idk if it's possible but try anyway
